backend/src/main/java/de/echochamber/backend/config/UserServiceConfigProperties.java

```diff
@@ -0,0 +1,19 @@
+package de.echochamber.backend.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "echochamber.user-service")
+public class UserServiceConfigProperties {
+
+    private String avatarUrl;
+
+    public String getAvatarUrl() {
+        return avatarUrl;
+    }
+
+    public void setAvatarUrl(String avatarUrl) {
+        this.avatarUrl = avatarUrl;
+    }
+}
```

Comment:
ich nehme an du hast die avatarUrl in die yaml ausgelagert? 
warum scheibst du sie nicht in die datenbank?